modules/statistics/php/stats_demographic.class.inc

```diff
@@ -32,21 +32,22 @@ class Stats_Demographic extends \NDB_Form
      */
     function _hasAccess()
     {
-        $user =& \User::singleton();
+        $user = \User::singleton();
+        //TODO: Create a permission specific to statistics
         return $user->hasPermission('data_entry');
     }
     /**
      * InCenter function
      *
-     * @param string $centerID the value of centreID
-     * @param string $Centres  the value of Centres
+     * @param string $centerID the value of centerID
```

Comment:
Do we use US English or Canadian (UK) English in our spelling?
I mostly saw Canadian spelling elsewhere (Code, poster, ...)

Reply:
I'm not sure, I originally added this change when I was reimplementing @johnsaigle's original PR. @driusan, thoughts?

Reply:
@samirdas used to insist on canadian spelling, but I don't think he's cared recently. I don't think "Center" has ever been spelled with Canadian spelling in LORIS, though.

Try bringing it up in a LORIS meeting.

Reply:
Putted it on the next Loris meeting agenda

Reply:
A quick grep in the code reveal 874 center vs 20 centre. Note that it include centered (at least once).
I think it is revealing. But still will bring it up at the meeting.

Reply:
My original goal was just to make the spelling consistent but I have no preference which one we use!

Reply:
I just want that we use a consistent language, not a mix of everything.
It make Loris look better (in my humble opinion).

Reply:
@PapillonMcGill I completely agree. After tomorrow's discussion I'll do a `:%s...` to whatever we decide.

Reply:
@ZainVirani Just be carefull, there are instance of string that contain `center` inside a word like `centered` or `docenter`. It might also be true with `centre`.

Reply:
removing all variable name, comments, ... I currently have 3 "center" and 1 "centre".